packages/amplify_auth_cognito/lib/amplify_auth_cognito.dart

```diff
@@ -121,20 +121,20 @@ class AmplifyAuthCognito extends AuthPluginInterface {
   }
 
   Future<UpdateUserAttributeResult> updateUserAttribute(
-      {@required UpdateUserAttributeRequest request}) async {
+      {UpdateUserAttributeRequest? request}) async {
```

Comment:
I think this should be required? I see all the other requests are also optional so maybe I am missing something. I can't think of a scenario in which this should be optional though.

Reply:
@haverchuck can you add your insights here?

Reply:
Yeah I would think this is required, since the customer has to pass the attribute-to-be-updated.